<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconstructionClass</class>
 <widget class="QMainWindow" name="ReconstructionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>960</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>960</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>.苹方-简</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>3D-Reconstruction</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>851</y>
      <width>1400</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.苹方-简</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>431</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>231</width>
       <height>391</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>319</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>滤波</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>离散傅里叶反变换</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>离散傅里叶变换</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>求相位</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>解相位</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>重建三维模型</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>319</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>One Step</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>图像处理操作</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>调制光栅图像处理结果</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>660</width>
      <height>380</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>原图</string>
     </attribute>
     <widget class="QLabel" name="labelImg1_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>频域图像</string>
     </attribute>
     <widget class="QLabel" name="labelImg1_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>滤波后频域图像</string>
     </attribute>
     <widget class="QLabel" name="labelImg1_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>滤波后空域图像</string>
     </attribute>
     <widget class="QLabel" name="labelImg1_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>包裹相位图</string>
     </attribute>
     <widget class="QLabel" name="labelImg1_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_12">
     <attribute name="title">
      <string>解包裹相位图</string>
     </attribute>
     <widget class="QLabel" name="labelImg1_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>参考平面图像处理结果</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>660</width>
      <height>380</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_14">
     <attribute name="title">
      <string>原图</string>
     </attribute>
     <widget class="QLabel" name="labelImg2_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_15">
     <attribute name="title">
      <string>频域图像</string>
     </attribute>
     <widget class="QLabel" name="labelImg2_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_16">
     <attribute name="title">
      <string>滤波后频域图像</string>
     </attribute>
     <widget class="QLabel" name="labelImg2_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_17">
     <attribute name="title">
      <string>滤波后空域图像</string>
     </attribute>
     <widget class="QLabel" name="labelImg2_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_18">
     <attribute name="title">
      <string>包裹相位图</string>
     </attribute>
     <widget class="QLabel" name="labelImg2_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_19">
     <attribute name="title">
      <string>解包裹相位图</string>
     </attribute>
     <widget class="QLabel" name="labelImg2_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>600</width>
        <height>340</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>450</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据统计</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>40</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>540</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>550</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前点坐标(mm)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>600</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前点高度(mm)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>640</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>650</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>待测物体体积(mm^3)：</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>850</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>20</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuswitchTo">
    <property name="title">
     <string>SwitchTo</string>
    </property>
    <addaction name="actionPMP"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuswitchTo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPMP"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 9pt &quot;.苹方-简&quot;;</string>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPMP">
   <property name="text">
    <string>PMP</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Reconstruction.qrc"/>
 </resources>
 <connections/>
</ui>
